scripts/sct_label_utils.py

```diff
@@ -681,195 +683,243 @@ def get_parser():
-                      mandatory=False)
-    parser.add_option(name='-keep',
-                      type_value=[[','], 'int'],
-                      description='Keep labels of specific value (specified here) from reference image',
-                      mandatory=False)
-
-    parser.usage.addSection("MISC")
-    parser.add_option(name="-msg",
-                      type_value="str",
-                      description='Display a message to explain the labeling task. Use with -create-viewer.',
-                      mandatory=False)
-    parser.add_option(name="-o",
-                      type_value=[[','], "file_output"],
-                      description="Output image(s).",
-                      mandatory=False,
-                      example="t2_labels_cross.nii.gz",
-                      default_value="labels.nii.gz")
-    parser.add_option(name="-v",
-                      type_value="multiple_choice",
-                      description='Verbose. 0: nothing. 1: basic. 2: extended.',
-                      mandatory=False,
-                      default_value=param_default.verbose,
-                      example=['0', '1', '2'])
-    parser.add_option(name='-qc',
-                      type_value='folder_creation',
-                      description='The path where the quality control generated content will be saved',
-                      default_value=None)
-    parser.add_option(name='-qc-dataset',
-                      type_value='str',
-                      description='If provided, this string will be mentioned in the QC report as the dataset the process was run on',
-                      )
-    parser.add_option(name='-qc-subject',
-                      type_value='str',
-                      description='If provided, this string will be mentioned in the QC report as the subject the process was run on',
-                      )
+    parser = argparse.ArgumentParser(
+        description="Utility function for label images.",
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help="Input image. Example: t2_labels.nii.gz"
+    )
+
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit."
+    )
+    optional.add_argument(
+        '-add',
+        metavar=Metavar.int,
+        type=int,
+        help="Add value to all labels. Value can be negative."
+    )
+    optional.add_argument(
+        '-create',
+        metavar=Metavar.list,
+        type=list_type(':', Coordinate),
+        help=("Create labels in a new image. List labels as: x1,y1,z1,value1:x2,y2,z2,value2."
```

Comment:
```suggestion
        help=("Create labels in a new image. List labels as: x1,y1,z1,value1:x2,y2,z2,value2. "
```

Reply:
I'm not sure I understand this suggestion. Could you clarify?

Reply:
Just to add a space between the period and the continuation on the next line so it's `value2. Example` not `value2.Example`

Reply:
Ah! Of course. :stuck_out_tongue_closed_eyes: